editor.planx.uk/docs/adding-a-new-component.md

```diff
@@ -1,10 +1,18 @@
-# Adding a new component
+# Adding a new @planx component
 
 Let's add a `SetValue` component
 
-`mkdir src/@planx/components/SetValue`
+## Core directory & files 
 
-`model.ts`
+1. `src/@planx/components/types.ts`
+
+```typescript
+SetValue = 380,
+```
+
+1. `mkdir src/@planx/components/SetValue`
```

Comment:
nit: Markdown auto incrementing fails when there's a block between points, making this document actually display as 1., 1., 1. etc.

Reply:
you're right - thanks for catching that ! manually numbered now.